aidial_adapter_openai/app.py

```diff
@@ -68,202 +26,30 @@ async def lifespan(app: FastAPI):
     await get_http_client().aclose()
 
 
-app = FastAPI(lifespan=lifespan)
-
-
-init_telemetry(app, TelemetryConfig())
-configure_loggers()
-
-
-def get_api_version(request: Request):
-    api_version = request.query_params.get("api-version", "")
-    api_version = API_VERSIONS_MAPPING.get(api_version, api_version)
+def create_app(
+    app_config: ApplicationConfig | None = None,
+    to_init_telemetry: bool = True,
```

Comment:
`to_do_something` for a boolean flag is a very unorthodox naming convention.

How about simply `init_telemetry`? You could avoid conflict using qualified imports:

```python
from aidial_sdk.telemetry.init import init_telemetry as sdk_init_telemetry
```

Reply:
Fixed